upgrade/UPGRADE-v8.1.0-dev.md

```diff
@@ -731,6 +731,14 @@ There you can find links to upgrade notes for other versions too.
         ```
 
 - add image and iframe LazyLoad ([#1483](https://github.com/shopsys/shopsys/pull/1483))
+    - lazy load is by default disabled, you can enable it by configuring new parameter in [`parameters_common`](https://github.com/shopsys/shopsys/blob/8.1/project-base/app/config/parameters_common.yml)
+        ```diff
+                shopsys.display_timezone: Europe/Prague
+        +       shopsys.image.enable_lazy_load: true
+        ```
+        !!! note
+            lazy loading will be enabled by default in next major
```

Comment:
Admonitions can't be used in upgrade notes as they're not rendered in documentation website and github markdown don't know them